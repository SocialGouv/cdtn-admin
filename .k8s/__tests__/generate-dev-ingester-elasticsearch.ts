//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/gitlab-ci.env";

jest.setTimeout(1000 * 60);
test("INGESTER_ELASTICSEARCH_TARGET=dev kosko generate --env dev jobs/ingester-elasticsearch", async () => {
  process.env.HARBOR_PROJECT = "cdtn";
  expect(
    await getEnvManifests("dev", "jobs/ingester-elasticsearch", {
      ...project("cdtn-admin").dev,
      INGESTER_ELASTICSEARCH_TARGET: "dev",
      RANCHER_PROJECT_ID: "c-bar:p-foo",
    })
  ).toMatchSnapshot();
});
