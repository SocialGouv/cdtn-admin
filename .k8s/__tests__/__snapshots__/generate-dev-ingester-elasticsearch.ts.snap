// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kosko generate ingester-elasticsearch --dev 1`] = `
"---
apiVersion: v1
kind: ConfigMap
metadata:
  name: ingester-elasticsearch
data:
  CDTN_ADMIN_ENDPOINT: http://hasura/v1/graphql
  NLP_URL: https://preprod-serving-ml.dev2.fabrique.social.gouv.fr
  NODE_ENV: production
---
apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  annotations:
    sealedsecrets.bitnami.com/cluster-wide: 'true'
  name: ingester-elasticsearch
  namespace: null
spec:
  encryptedData:
    ELASTICSEARCH_DATA_TOKEN: >-
      AgA+5TqL1kYeeCl8R7j0JE6dgEQ+9gNtGvD2eGyNHYveWA9N0l3UD+EF3OUNzcoeBseMVK/7tgPC1PBXTluABbIMZe8v0oKf7McgJZEkXpzUh/wL+icjGKIDb1zFCYV+Ox2m1L0fsMVRcOEBAh0RpgRcY8CEpLpszoEvlgTKnFljkqQfDUTwofr96Q6FYolD1cjikx+jzBvhxAkbf8mrrwz775UUY3u9OPFk3TQSqVVyd0kzwJWRTRlb1g3mYtfC79FM1j/44LkBYdeAQfNDTx12LNg+n23RQLXlGfEu2oTMfkVQgVnQ+Xqxgzc4wvvglgOP+yJe0S01i49uBXIh3ZniBM99rVtwRONu6Q1RTN2lzGwdMPT1WhiCk8nJDqrLFYs2yg++qWYo/HMPVDYCbzpH+7j1NbEvzUSH4+KSb8Qsk1YIIMskOACWNpvmAraT6KzxsK/QnEAQckex5Y03xSFB1L4boAlVkmhOkJXaz92Ug/ab6tMWTKoalz0zxBYHjVovJeN0RhKV9kw48S12B3aSkqgO3QTHN7CY/FsebcUZgE2eup+LDyPRwM1IMwMsU+t8QL73YBdZx6Fk01nBBuT3D5Ba46xA4dWm4tLH04niErwhuSJalaHQA3AwUNdAiUx9tf48lkAeinALlcGG9Hj1o+EqhF9dXFad+4QQUN7nXIkSLRpEu9kWQZKW2FnlfJ6KADkokMmm77eMFAL8ifToozRn7rgjopyaR9hVxT91iNmEhWpKopMBfRjzP/bP9uWMr7cqIwLQlKAUgwg=
    ELASTICSEARCH_URL: >-
      AgBxkNsQpkk6V5GIA8WuYBXE1iTLglrzRoR5HQ9QtBzsjDb9siCHQc+U9UFI72hDKi3yNO2g1lk+Nh7ooKHpqzzV0PSB0gyUJwo4MmqfxUDw3ef2BHezK7DK0z1YB/ZU8BLK5LenLchoKsKejv1YC1OQmG6EUHPW/3hm2qiS56anl6AIyUqXsokx+5QScjmHE3KybThaBkva/7PEQn/vks9wbY2mePvBPanSnVSLuSUY0yn0UXrmJnyW87oXVaJSQU/+5oDPdnb4pN8wivmjQEBLyVvjL2xSFz2druu4wjDpArup981GA7jCTarCzxjomGKnfxu7U98YKho1OUbuGevL1TXZaOkXaSIpls5mViZjUB2WaUVqQhHr9K1f6Hmhidh6S7v8NX4NbPKgKA7PTJ1dzi7EeKUCiIJPXiSI76H/rNlEx55TlroXQxEPjmQ91yEcCtRNa6MS2X6Flu539+t2m0+nm+Wqx7SmbGROjvCbSaYMShdsYQgnRj3VMgzoToLcnRPO0MdIop2L6GN5ocyFPk/QRUry+A/TH5eYkuMGu/FhctdrNEMo2kcYgIiTfqQ9tuQQqD3UCJ74DkUOod8OJoTu2EQ6RVBXqVCb9cM0AmoYgUDx1JW0l1124UYzpqIPhaRdPe6M8JH9zXuQAI93R7Paezn8GEMaXfiBRc/fVRUdTbAdbGvdmEZVeiczEK+JsKcdvMuqud15UmcFF0N7rfuia1u6TjCvaF/gvFIlLnMccFYmKACJnf1SJ/5Cx6L0SajjyBhxNP8DMVHhmMpbIqfQBPJho0nVQk7ZoU5Heg==
    ES_LOGS_TOKEN: >-
      AgA++SZAeq0DcFzNosxv9AyBgnXWQegUmfQlSQ9z8RlUjPxgAdLIGAMFjtL5gPu3CFIiR48ELkWq6pEqWvMypp5wTUrNwZsjktPYh49v96wW8KCRTGMtR9MbI8x52DKPl4KNAGzOixObi+8qP4+4eku6KVgOajsvAS1zJSQ+R1tJAKLBfIEhebOScmtaXJWty3voOVsORPg0tkrxWxK2+1iPkYV6sL6fsK04yRnbeBY6ULz0SNgba9CMePCNgOBgbIbsYR7Mg50dL6N0V9ttg00r/FP71ahQUp7R4TcJZfFrF84H6RxXvbfXfjjdQYQsJW4yHuHybFFBleQ7KLnQ33CgAh7oQ9pzS+wOBGnFQOylq57kVxB28VqpfZnybDVfz3LaW4QNLlbaFL/rnwYuCpNN+UQ0EgtC7a+i76t9z80xzhvTsSJDodHOdm8YbeWvol3V7mI2EO60D5LaXnMUd9txEiyL3HZ/hfsOkCXtx6uwuzgxEhMVQUEhydMG0UYWmcR7uB1CE9RLLqMEj4XdRlEQvH5Sf3YjLgnLFRpZtdi7gk8ifLdK1Sv0u34GgSzTIE5gMkfE7SQyFQem/N0BlNLPkLe3J1d7lIlGHxD64fN0MgcEl+cKTrqnXJS66I76p/RUR5Tkrz3L0T2neIS75PCPLctA3f5cQodH30rZfIOT+xwb6pfKFUuOV4NCwQ0QUlJ/QUslrTnnbC0dOQxZ0gJysBEpiGO8uIqfhrL3k/zKQ+WggeB/SHn7ol0GyleT/BEKwnGRBR98r1T7rYo=
    ES_LOGS: >-
      AgCbpO0UoNnt2w12E+roWCetKyHKnfiUxfeSQhY3TRi+FSzR5Tcq8WCE7w/Onp0djnpYYp6edWi45InhJU9j9TFE/I9lw0/0EiNX7uiS7kaJkwWFZi0cJXDlPbj2zc/aHeuwzoYpS7PeKT24WHZ/UycSl9Ce59cUpK6kHIieD44HgT3N/+EYR9ihX1Ibu5a0PTuQfeVdEOGYZwG+fNaUvkHr955crrxMakxwLKNz5uMLNGTYuE4LFreDe/owLEK13/nt0/oAMUXotZpvm7f/pG+xTk6+kQIRjFK+Qr8xBVzdQP9wSqHR/+J/GR4wJ7ap40UhBALV/w0rbAttjN/SL9C7kLUWgkV11wcAlcVP1yT46BK7uSms9v3SgmQJJN1TCR16K1pP+Ic/xD2oalkjEkcjrxddoRV1fm+Lnr+kVD3eoalR0Fw8tqd9GfIZv2EjNpKM2+Jll4aXBBiXIjfqbxWKRpFZvENi9OK3CR440yikw/6dkhdr7GCfmqUu8VzW/gegsiuAaGQOQqjc0sUocKOZ40d7adf0XCgqZp5qRYdVhYcwxHEnOJXkGn+k9Jm2CIrB5ZgCsmD9tdWwX9DiS6AGID9Co7xzSehF9ODh/vneRGltGbY2Ug3u9UH0FX4xlr3hJMUBA5v2dX7oYCa/gTEPghXOAuhohBgvLYFuULuXNXQgPbA9Wk6XchGJMkSZ288OIYriaJ6Vf6pw7KUvL6Sa4U4S88Sy7kJY1i1iu0weVG+qdUzBA/Zb3Arq7/VYW/F8vp8OqBqlAHywd3qRdf3rvV4Ld1dp/kCuQYnXpb7DZw==
  template:
    metadata:
      annotations:
        sealedsecrets.bitnami.com/cluster-wide: 'true'
      name: ingester-elasticsearch
    type: Opaque
---
apiVersion: batch/v1
kind: Job
metadata:
  name: ingester-elasticsearch
  namespace: cdtn-admin-85-master-dev2
spec:
  backoffLimit: 0
  template:
    spec:
      containers:
        - name: ingester-elasticsearch
          image: >-
            registry.gitlab.factory.social.gouv.fr/socialgouv/cdtn-admin/ingester-elasticsearch:0123456789abcdefghijklmnopqrstuvwxyz0123
          imagePullPolicy: IfNotPresent
          resources:
            limits:
              cpu: '2'
              memory: 1Gi
            requests:
              cpu: '1'
              memory: 512Mi
          envFrom:
            - configMapRef:
                name: ingester-elasticsearch
            - secretRef:
                name: ingester-elasticsearch
          env:
            - name: ES_INDEX_PREFIX
              value: cdtn-master
      restartPolicy: Never
  ttlSecondsAfterFinished: 86400
"
`;
